Northstar.CustomServers/mod/scripts/vscripts/titan/_replacement_titans.gnut

```diff
@@ -174,6 +176,18 @@ function IsReplacementTitanAvailableForGameState()
 	return true
 }
 
+bool function IsReplacementTitanAvailableForGameMode( entity player )
+{
+	bool shouldDoReplacementTitan = true
+	string gameMode = GameRules_GetGameMode()
+	if ( gameMode in file.onPlayerRequestTitanCallbacks )
+	{
+		foreach( callbackFunc in file.onPlayerRequestTitanCallbacks[ gameMode ] )
+			shouldDoReplacementTitan = callbackFunc( player )
```

Comment:
Should be `shouldDoReplacementTitan = shouldDoReplacementTitan & callbackFunc( player )`, otherwise only the last callback function ran determines the final result.